### PR TITLE
Update Podfile sources

### PR DESCRIPTION
**Pull request by jeffdp**
This change updates the podfile to use more specific sources before the general Cocoapod spec repo

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-source 'https://github.com/streem/cocoapods'
+source 'https://github.com/CocoaPods/Specs.git'
 
 use_frameworks!
 
 platform :ios, '11.0'
 
 target 'StreemNow' do
-  pod 'StreemKit', '~> 0.18.0'
+  pod 'StreemKit', '~> 0.18.0', :source => 'https://github.com/streem/cocoapods'
```
Comment (dgoldman-pdx) [inline Podfile:9]: Aren't we up to `0.19.0`?

**Review by dgoldman-pdx — APPROVED** [review]
okay w/ me!

File: Podfile (hunk 1 above)
Comment (jeffdp) [reply Podfile:9]: Looks like that change hasn't been merged back yet.

**Review by callum-code — APPROVED** [review]
💯

**Pull request merged**